feat(Curriculum) Add A1 Spanish Module 1

nieldakarla opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or Ona.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

<!-- Feel free to add any additional description of changes below this line -->

This PR adds all the tasks of the Chapter 1 module 1 of the A1 Spanish curriculum using the chapter based structure. 
Content of this module has been broken down into: 
Warm-up: Introduction of new words. (challengeType: challenge-grid)
Learn: Teaching words in context. (challengeType: challenge-grid)
Practice: Giving students the opportunity to practice what was covered in Learn (challengeType: challenge-grid)
Review: with Grammar highlights and Glossary (challengeType: link)
Quiz: 10 questions to assess knowledge. (challengeType: link)

nieldakarla committed:
implemented chapter structure and addressed coming soon error issue

nieldakarla committed:
changed the block layout

nieldakarla committed:
add first two tasks

nieldakarla committed:
add task 3

nieldakarla committed:
updated json

nieldakarla committed:
module 1 warm-up ready

nieldakarla committed:
fix some block 1 tasks

nieldakarla committed:
approve intro.json!

nieldakarla committed:
moved task 2 to block 2 and renamed tasks

nieldakarla committed:
add monologue 2

nieldakarla committed:
add monologue 3

nieldakarla committed:
fix some learn errors

nieldakarla committed:
finished practice section

nieldakarla committed:
add review and quiz

nieldakarla committed:
fix fill-in-the-blanks

nieldakarla committed:
fix tasak 2

nieldakarla committed:
fix some tasks

nieldakarla committed:
change to challenge-grid and fix punctuation

nieldakarla committed:
final adjustments

huyenltnguyen reviewed (commented):
I didn't review the PR in depth, just wanted to share a few typos I noticed while testing the branch locally :)

nieldakarla commented on curriculum/challenges/english/blocks/es-a1-learn-greetings-during-the-day/68c892f78e9dd27b55a0dc18.md:
```diff
@@ -0,0 +1,84 @@
+
+# --explanation--
+
+`Diseñadora` means "designer" in Spanish and is the **feminine form** of the noun `diseñador`.
+
+`Soy diseñadora UX` – I am a UX designer.
+
+This word includes the letter `ñ`, which is **not just an accented 'n'** — it is its own letter in the Spanish alphabet.  
+
+It sounds like "ny" in English (as in "canyon"). For example: 
```
```suggestion
`Diseñadora` means "designer" in Spanish and is the **feminine form** of the noun `diseñador`. For example: 

`Soy diseñadora UX` – I am a UX designer.

This word includes the letter `ñ`, which is **not just an accented 'n'** — it is its own letter in the Spanish alphabet.  

It sounds like "ny" in English (as in "canyon"). 
```

nieldakarla committed:
Apply suggestions from Huyen code review

Co-authored-by: Huyen Nguyen <25715018+huyenltnguyen@users.noreply.github.com>

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68c58ac706597fe1f643837d.md:
```diff
@@ -0,0 +1,105 @@
+
+## --answers--
+
+`Hasta luego.`
```
```suggestion
`Hasta luego`
```

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68c58ac706597fe1f643837d.md:
```diff
@@ -0,0 +1,105 @@
+
+---
+
+`Hasta pronto.`
```
```suggestion
`Hasta pronto`
```

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68c58ac706597fe1f643837d.md:
```diff
@@ -0,0 +1,105 @@
+
+---
+
+`Adiós.`
```
```suggestion
`Adiós`
```

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68c58ac706597fe1f643837d.md:
```diff
@@ -0,0 +1,105 @@
+
+---
+
+`Chao.`
```
```suggestion
`Chao`
```

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68c58b01490a81e467b6f8d8.md:
```diff
@@ -0,0 +1,103 @@
+
+## --answers--
+
+`Hasta mañana.`
```
```suggestion
`Hasta mañana`
```

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68c58b01490a81e467b6f8d8.md:
```diff
@@ -0,0 +1,103 @@
+
+---
+
+`Hasta luego.`
```
```suggestion
`Hasta luego`
```

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68c58b01490a81e467b6f8d8.md:
```diff
@@ -0,0 +1,103 @@
+
+---
+
+`Hasta pronto.`
```
```suggestion
`Hasta pronto`
```

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68c58b01490a81e467b6f8d8.md:
```diff
@@ -0,0 +1,103 @@
+
+---
+
+`Adiós.`
```
```suggestion
`Adiós`
```

estefaniacn reviewed (commented):
Hi, Nieda. The new structure looks great to me so far. This is my review for the tasks of the Warm-up section. I will continue reviewing the next sections.

estefaniacn reviewed (commented):
These are my thoughts and suggestions for the Learn section.

estefaniacn reviewed (commented):
These are my suggestions for the Practice section.

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c850ed23d494c9be5cb22a.md:
```diff
@@ -1,58 +1,87 @@
 # --description--
 
-Watch the video below to understand the context of the upcoming lessons.
+`¡Felicidades!` Congratulations on completing your first **Learn** and **Practice** sections!
```
```suggestion
`¡Felicitaciones!` Congratulations on completing your first **Learn** and **Practice** sections!
```
I would suggest using this word, since it's more focused on achievements.

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c850ed23d494c9be5cb22a.md:
```diff
@@ -1,58 +1,87 @@
+
+- `Soy Camila.` – I am Camila.  
+
+- `Me llamo Esteban.` – My name is Esteban.
+
+### Using `soy` to Talk About Identity
```
```suggestion
- `Me llamo Esteban.` – My name is Esteban.

<br />

### Using `soy` to Talk About Identity
```

estefaniacn replied:
I would suggest adding these between the subsections to add more vertical space.

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c850ed23d494c9be5cb22a.md:
```diff
@@ -1,58 +1,87 @@
+
+In English, people say "a developer", but in Spanish, you **don't use** an article (`un` or `una`) before a profession.
+
+Use `Soy analista`.
```
```suggestion
You can use `Soy analista`.
```
To keep the "you" tone from the previous paragraph.

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c850ed23d494c9be5cb22a.md:
```diff
@@ -1,58 +1,87 @@
+
+Accents in Spanish show where to put the stress and sometimes change the meaning of a word. Examples from this lesson:
+
+- `Días` – days (stress on the `í`)  
```
```suggestion
- `Días` – days (stress on the `í`).
```

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c850ed23d494c9be5cb22a.md:
```diff
@@ -1,58 +1,87 @@
+
+- `Días` – days (stress on the `í`)  
+
+- `Adiós` – goodbye (stress on the `ó`)  
```
```suggestion
- `Adiós` – goodbye (stress on the `ó`).
```

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c850ed23d494c9be5cb22a.md:
```diff
@@ -1,58 +1,87 @@
+
+`Me llamo` – my name is
+
+`Ñ` – This letter has a distinct sound, similar to the "ny" in "canyon". It is treated as a separate letter in the Spanish alphabet. For example: 
```
```suggestion
`Ñ` – This letter has a distinct sound. It is treated as a separate letter in the Spanish alphabet. For example: 
```
I would suggest removing the comparison because the sound is more pronounced in Spanish.

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c994e8517cd6bde390e923.md:
```diff
@@ -0,0 +1,53 @@
+
+- `Hola` – Hello  
+
+- `Me llamo` – My name is.  
```
```suggestion
- `Me llamo` – My name is
```

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c994e8517cd6bde390e923.md:
```diff
@@ -0,0 +1,53 @@
+
+- `Me llamo` – My name is.  
+
+- `Soy` – I am.
```
```suggestion
- `Soy` – I am
```

estefaniacn reviewed (commented):
These are my thoughts for the Review section.

estefaniacn reviewed (commented):
And these are my thoughts on the Quiz.

nieldakarla commented on curriculum/challenges/english/blocks/es-a1-learn-greetings-during-the-day/68c892f78e9dd27b55a0dc18.md:
```diff
@@ -0,0 +1,84 @@
+`Soy diseñadora UX` – I am a UX designer.
+
+This word includes the letter `ñ`, which is **not just an accented 'n'** — it is its own letter in the Spanish alphabet.  
+
+It sounds like "ny" in English (as in "canyon"). 
```
```suggestion
```

nieldakarla commented on curriculum/challenges/english/blocks/es-a1-learn-greetings-during-the-day/68c896d45893e91d602f0373.md:
```diff
@@ -0,0 +1,104 @@
+
+### --feedback--
+
+This greeting is used earlier in the day, not in the evening.
```
```suggestion
The greeting Camila is saying is used earlier in the day, not in the evening.
```

nieldakarla commented on curriculum/challenges/english/blocks/es-a1-learn-greetings-during-the-day/68c896d45893e91d602f0373.md:
```diff
@@ -0,0 +1,104 @@
+
+### --feedback--
+
+The speaker is not saying "good night". Listen again carefully.
```
```suggestion
The speaker is not saying "good night".
```

nieldakarla commented on curriculum/challenges/english/blocks/es-a1-learn-greetings-during-the-day/68c897fe7948d82ccadf1f3b.md:
```diff
@@ -0,0 +1,105 @@
+
+### --feedback--
+
+This means "designer", but it's not the profession mentioned. Pay attention to the last word.
```
```suggestion
This means "designer", but it's not the profession mentioned. Pay attention to the last word of the audio.
```

nieldakarla commented on curriculum/challenges/english/blocks/es-a1-learn-greetings-during-the-day/68c897fe7948d82ccadf1f3b.md:
```diff
@@ -0,0 +1,105 @@
+
+### --feedback--
+
+This means "engineer", which is different from what the speaker says. Pay attention to the last word.
```
```suggestion
This means "engineer", which is different from what the speaker says. Pay attention to the last word of the audio.
```

nieldakarla commented on curriculum/challenges/english/blocks/es-a1-practice-greetings-and-farewells/68c9881715d314368f261622.md:
```diff
@@ -0,0 +1,141 @@
+
+### --feedback--
+
+This greeting is used earlier in the day, not in the evening.
```
```suggestion
The greeting Camila says used earlier in the day, not in the evening.
```

nieldakarla commented on curriculum/challenges/english/blocks/es-a1-practice-greetings-and-farewells/68c98ad382b60c54f0d4d6bf.md:
```diff
@@ -0,0 +1,77 @@
+
+Camila is introducing herself at work in the afternoon. Complete her introduction with the correct words.
+
+Use the words from the list below. Some words won't be used.
```
Perfect!

nieldakarla commented on curriculum/challenges/english/blocks/es-a1-quiz-greetings-and-farewells/68c8510f4a9c00ca2a67160c.md:
```diff
@@ -1,58 +1,239 @@
+
+This quiz checks your understanding of greetings, farewells, and basic introductions in Spanish.  
+
+There is no audio, so read each question carefully and choose the correct answer.
```
```suggestion
Read each question and choose the correct answer. There's just one correct answer for each question.
```

nieldakarla committed:
Apply suggestions from Estefania's code review

I've added almost all suggestions with exception of some punctuation and the usage of "please". These should be discussed during our meeting.

Co-authored-by: Estefania Cassingena Navone <85124911+estefaniacn@users.noreply.github.com>

nieldakarla committed:
Apply suggestions from code review

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-learn-greetings-during-the-day/68c89d77ea2edc6d84388cb7.md:
```diff
@@ -0,0 +1,106 @@
+
+### --feedback--
+
+This phrase has **two words** and means "See you tomorrow". It's used when you expect to see the person the next day. Remember to use the letter `ñ` in the second word.
```
```suggestion
This phrase has **two words** and means "See you tomorrow". It's used when you expect to see the person the next day. The first letter of the first word is capitalized, and remember to use the letter `ñ` in the second word.
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-practice-greetings-and-farewells/68c98887b9a4783bf6788700.md:
```diff
@@ -0,0 +1,125 @@
+
+### --feedback--
+
+He says "Hasta mañana".
```
```suggestion
He says `Hasta mañana`.
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-practice-greetings-and-farewells/68c98a694e038945907506b2.md:
```diff
@@ -0,0 +1,60 @@
+
+# --explanation--
+
+Both `Me llamo` and `Soy` can be used to introduce yourself in Spanish.
+
+- `Me llamo Camila` – My name is Camila  
+
+- `Soy Camila` – I am Camila  
+
+Both are correct, but `Me llamo` is more common when saying your name for the first time.
```
```suggestion
Both `Me llamo...` and `Soy...` can be used to introduce yourself in Spanish.

- `Me llamo Camila.` – My name is Camila.

- `Soy Camila.` – I am Camila.

Both are correct, but `Me llamo...` is more common when saying your name for the first time.
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-practice-greetings-and-farewells/68c98ad382b60c54f0d4d6bf.md:
```diff
@@ -0,0 +1,77 @@
+
+### --feedback--
+
+This means "later" and is used in the farewell "Hasta luego".
```
```suggestion
This means "later" and is used in the farewell.
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-practice-greetings-and-farewells/68c98ad382b60c54f0d4d6bf.md:
```diff
@@ -0,0 +1,77 @@
+
+# --explanation--
+
+This short introduction uses the greeting `Hola`, the afternoon phrase `buenas tardes`, the name phrase `Me llamo`, the verb `Soy` for the profession, and the farewells `Hasta luego` and `Adiós`.
```
```suggestion
This short introduction uses the greeting `Hola`, the afternoon phrase `Buenas tardes`, the name phrase `Me llamo`, the verb `Soy` for the profession, and the farewells `Hasta luego` and `Adiós`.
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-practice-greetings-and-farewells/68c98ca32c262266f70e198c.md:
```diff
@@ -0,0 +1,64 @@
+
+### --feedback--
+
+Only one of these matches correctly. Please listen again or check your notes.
```
```suggestion
Only one of these matches correctly. Please review each farewell.
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-practice-greetings-and-farewells/68c98ca32c262266f70e198c.md:
```diff
@@ -0,0 +1,64 @@
+
+Here are the correct translations:
+
+- `Adiós` – Goodbye  
+
+- `Hasta luego` – See you later 
+
+- `Hasta mañana` – See you tomorrow  
+
+- `Hasta pronto` – See you soon  
```
```suggestion
- `Adiós.` – Goodbye.

- `Hasta luego.` – See you later. 

- `Hasta mañana.` – See you tomorrow.

- `Hasta pronto.` – See you soon.
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-quiz-greetings-and-farewells/68c8510f4a9c00ca2a67160c.md:
```diff
@@ -1,58 +1,239 @@
+
+This quiz checks your understanding of greetings, farewells, and basic introductions in Spanish.  
+
+Read each question and choose the correct answer. There's just one correct answer for each question.
```
```suggestion
Read each question and choose the correct answer. There's only one correct answer for each question.
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-quiz-greetings-and-farewells/68c8510f4a9c00ca2a67160c.md:
```diff
@@ -1,58 +1,239 @@
+
+#### --text--
+
+Which phrase means “See you later”?
```
```suggestion
Which phrase means "See you later"?
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-quiz-greetings-and-farewells/68c8510f4a9c00ca2a67160c.md:
```diff
@@ -1,58 +1,239 @@
+
+#### --text--
+
+Which phrase means “Good night / Good evening”?
```
```suggestion
Which phrase means "Good night / Good evening"?
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-quiz-greetings-and-farewells/68c8510f4a9c00ca2a67160c.md:
```diff
@@ -1,58 +1,239 @@
+
+#### --text--
+
+Which phrase means “See you tomorrow”?
```
```suggestion
Which phrase means "See you tomorrow"?
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-quiz-greetings-and-farewells/68c8510f4a9c00ca2a67160c.md:
```diff
@@ -1,58 +1,239 @@
+
+#### --text--
+
+Which phrase means “I am a designer”?
```
```suggestion
Which phrase means "I am a designer"?
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-quiz-greetings-and-farewells/68c8510f4a9c00ca2a67160c.md:
```diff
@@ -1,58 +1,239 @@
+
+#### --text--
+
+What is the correct way to say “I'm an analyst”?
```
```suggestion
What is the correct way to say "I'm an analyst"?
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-quiz-greetings-and-farewells/68c8510f4a9c00ca2a67160c.md:
```diff
@@ -1,58 +1,239 @@
+
+#### --distractors--
+
+`Soy` / I call myself.
```
This might be confusing as it was taught that `Soy` can be used to introduce your name

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c850ed23d494c9be5cb22a.md:
```diff
@@ -1,58 +1,89 @@
+
+### Using `soy` to Talk About Identity
+
+The verb `ser` (to be) is used to describe who you are; Like your name or profession.
```
```suggestion
The verb `ser` (to be) is used to describe who you are; like your name or profession.
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c850ed23d494c9be5cb22a.md:
```diff
@@ -1,58 +1,89 @@
+
+### Special Letters in Spanish
+
+`ll` – This double letter is usually pronounced like a `y` in many Spanish-speaking countries. For example: 
```
```suggestion
`ll` – This double letter is usually pronounced like a `y` in many Spanish-speaking countries. For example: 
```
There's a localization concern on this part: is usually pronounced like a `y`

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c850ed23d494c9be5cb22a.md:
```diff
@@ -1,58 +1,89 @@
+
+`ll` – This double letter is usually pronounced like a `y` in many Spanish-speaking countries. For example: 
+
+`Me llamo` – my name is
```
```suggestion
`Me llamo...` – my name is...
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c994e8517cd6bde390e923.md:
```diff
@@ -0,0 +1,53 @@
+
+## Greetings and Introductions
+
+- `Buenas noches` – Good evening / Good night  
+
+- `Buenas tardes` – Good afternoon  
+
+- `Buenos días` – Good morning  
+
+- `Hola` – Hello  
+
+- `Me llamo` – My name is.  
+
+- `Soy` – I am.
+
+<br>
+
+## Farewells
+
+- `Adiós` – Goodbye  
+
+- `Chao` – Bye  
+
+- `Hasta luego` – See you later  
+
+- `Hasta mañana` – See you tomorrow  
+
+- `Hasta pronto` – See you soon
```
```suggestion
- `Buenas noches.` – Good evening / Good night.  

- `Buenas tardes.` – Good afternoon.  

- `Buenos días.` – Good morning.  

- `Hola.` – Hello.  

- `Me llamo...` – My name is...

- `Soy...` – I am...

<br>

## Farewells

- `Adiós.` – Goodbye.  

- `Chao.` – Bye.  

- `Hasta luego.` – See you later.  

- `Hasta mañana.` – See you tomorrow.  

- `Hasta pronto.` – See you soon.
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68bb001c22f3bcbd1fdf38fa.md:
```diff
@@ -14,14 +14,14 @@ Welcome to the A1 Professional Spanish course!
 
 Listen to the audio and answer the question below.
-There's just **one** correct answer. If you need help, click on the `explanation` at the bottom of the lesson.
+There's just **one** correct answer. If you need help, click on **explanation** at the bottom of the lesson.
```
```suggestion
There's only **one** correct answer. If you need help, click on the **explanation** at the bottom of the lesson.
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68bb001c22f3bcbd1fdf38fa.md:
```diff
@@ -65,7 +65,7 @@ This is a polite word, but it's not what you hear in the audio.
 
 The word `Hola` is a greeting. It is used when you meet someone, similar to "Hello" or "Hi" in English. For example:
```
```suggestion
The word `Hola` is a greeting. It is used when you meet someone, similar to "Hello" or "Hi". For example:
```

miyaliu666 replied:
```suggestion
The word `Hola` is a greeting. It is used when you meet someone. For example:
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68c582b401546d5359f54658.md:
```diff
@@ -0,0 +1,104 @@
+
+# --explanation--
+
+`Buenos días` is a common greeting used in the morning.  
+It literally means "good days", but it's equivalent to "good morning" in English. For example:  
+
+`Buenos días, ¿Cómo estás?` – Good morning, how are you?
```
```suggestion
`Buenos días` is a common greeting used in the morning. It literally means "good days", but it's equivalent to "good morning". For example:  

`Buenos días, ¿Cómo estás?` – Good morning, how are you?
```
Is the punctuation in the example correct?

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68c5877f0c90f5b8cad8e2dd.md:
```diff
@@ -0,0 +1,107 @@
+
+# --explanation--
+
+`Buenas noches` is a greeting used at night.  
+It translates to "Good night".
+
+In Spanish, you can use `Buenas noches` both when greeting someone in the evening **and** when saying goodbye at night. For example:  
+
+`Buenas noches, señora López.` – A way to greet or say goodbye in the evening.
```
```suggestion
`Buenas noches` is a greeting used at night. It translates to "Good night".

In Spanish, you can use `Buenas noches` both when greeting someone in the evening **and** when saying goodbye at night. For example:  

`Buenas noches, señora López.` – A way to greet or say goodbye in the evening.
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68c586973237a2ae8b8a9dcb.md:
```diff
@@ -0,0 +1,104 @@
+
+# --explanation--
+
+`Buenas tardes` is a greeting used during the afternoon.  
+It translates to "Good afternoon". For example:
```
```suggestion
`Buenas tardes` is a greeting used during the afternoon. It translates to "Good afternoon". For example:
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68c5893734d89dcb8e5b7a34.md:
```diff
@@ -0,0 +1,106 @@
+
+### --feedback--
+
+This means "See you soon", and it's similar to "Hasta luego", but it's not the correct word here.
```
```suggestion
This means "See you soon", and it's similar to `Hasta luego`, but it's not the correct word here.
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68c5893734d89dcb8e5b7a34.md:
```diff
@@ -0,0 +1,106 @@
+
+# --explanation--
+
+`Adiós` is a common way to say goodbye in Spanish.  
+It translates to "Goodbye".
```
```suggestion
`Adiós` is a common way to say goodbye in Spanish. It translates to "Goodbye".
```

anastasiiauk commented on curriculum/challenges/english/blocks/es-a1-warm-up-greetings-and-farewells-basics/68c58aa8c9d2a2dfea45deb7.md:
```diff
@@ -0,0 +1,106 @@
+
+# --explanation--
+
+`Hasta luego` is a common way to say goodbye in Spanish.  
+It translates to "See you later".
```
```suggestion
`Hasta luego` is a common way to say goodbye in Spanish. It translates to "See you later".
```

miyaliu666 commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c850ed23d494c9be5cb22a.md:
```diff
@@ -1,58 +1,89 @@
+
+### No Articles Before Professions
+
+In English, people say "a developer", but in Spanish, you **don't use** an article (`un` or `una`) before a profession.
```
This certification will be translated into different languages, right? Considering that some learners may not have learned English, we'd better avoid using "in English".

nieldakarla committed:
Apply 2 suggestions from Anastasiia to test it

Co-authored-by: Anastasiia <109740924+anastasiiauk@users.noreply.github.com>

nieldakarla commented on curriculum/challenges/english/blocks/es-a1-learn-greetings-during-the-day/68c89d77ea2edc6d84388cb7.md:
```diff
@@ -0,0 +1,106 @@
+
+`Hasta mañana` means "See you tomorrow". It's used when you expect to see the person the next day.
+
+`Chao` is an informal version of "goodbye" used in some Spanish speaking countries. It is similar to "bye" in English. For example:  
```
> 1. `similar to "bye" in English` is confusing (even for me, an English learner). Does it mean `bye` is the informal version of `goodbye`?
> 2. Should we strictly avoid using `in English`?

I think we should discuss how this would go in translations. I'm more inclined to avoid comparing it to English altogether. But I'm not 100% sure yet.

nieldakarla committed:
Apply suggestions from code review

Co-authored-by: Anastasiia <109740924+anastasiiauk@users.noreply.github.com>

nieldakarla committed:
Apply suggestions from code review

Co-authored-by: Anastasiia <109740924+anastasiiauk@users.noreply.github.com>

nieldakarla committed:
Apply suggestions from code review

Co-authored-by: Anastasiia <109740924+anastasiiauk@users.noreply.github.com>

nieldakarla committed:
Apply suggestions from code review

Co-authored-by: Anastasiia <109740924+anastasiiauk@users.noreply.github.com>

nieldakarla committed:
review warm-up and add description

nieldakarla committed:
first 5 tasks of learn

nieldakarla committed:
review learn block

nieldakarla committed:
review practice

nieldakarla committed:
review review section

nieldakarla committed:
review quiz

nieldakarla committed:
fix comment space

nieldakarla committed:
fix extra blank

nieldakarla committed:
fix reviews

estefaniacn reviewed (commented):
Thank you for making these updates from my previous review, Nielda. These are my thoughts and suggestions for the new version.

nieldakarla commented on curriculum/challenges/english/blocks/es-a1-learn-greetings-during-the-day/68c893dd7bbd1e85869d0e5b.md:
```diff
@@ -0,0 +1,106 @@
+
+# --description--
+
+There are different ways to say goodbye in Spanish. In this task you'll hear two common farewell expressions that are often used together. One means "see you later" and the other means "goodbye".
```
```suggestion
There are different ways to say goodbye in Spanish. In this task you'll hear two common farewell expressions that are often used together. 

One means "see you later" and the other means "goodbye".
```

nieldakarla commented on curriculum/challenges/english/blocks/es-a1-practice-greetings-and-farewells/68c98a694e038945907506b2.md:
```diff
@@ -0,0 +1,64 @@
+
+## --text--
+
+Which of the following options could you use to introduce your name?
```
```suggestion
Which of the following options contain two different ways of introducing your name in Spanish?
```

nieldakarla commented on curriculum/challenges/english/blocks/es-a1-practice-greetings-and-farewells/68c98ad382b60c54f0d4d6bf.md:
```diff
@@ -0,0 +1,81 @@
+
+**IMPORTANT**: Some words won't be used, others simply don't make sense in context, and you can only use each word **once**.
+
+`Hola`, `Me llamo`, `Soy`, `Adiós`, `luego`, `tardes`, `noches`, `mañana`.
```
@estefaniacn this is a good question. In the English version we just capitalize them if they are capitalized in the original text. I'll keep them this way for now and in the future we might change that.

nieldakarla commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c850ed23d494c9be5cb22a.md:
```diff
@@ -1,58 +1,87 @@
+
+### Greeting with Time of Day
+
+Spanish greetings change depending on the time of day. They are treated as **set phrases**. For example:
```
```suggestion
Spanish greetings change depending on the time of day. For example:
```

nieldakarla commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c850ed23d494c9be5cb22a.md:
```diff
@@ -1,58 +1,87 @@
+
+### Special Letters in Spanish
+
+`ll` in the word `llamo`. For example:
```
@estefaniacn should we remove this, then?

nieldakarla committed:
Apply suggestions from Estefania

Co-authored-by: Estefania Cassingena Navone <85124911+estefaniacn@users.noreply.github.com>

nieldakarla committed:
add Estefania's suggestions

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-practice-greetings-and-farewells/68c98a694e038945907506b2.md:
```diff
@@ -0,0 +1,64 @@
+
+## --text--
+
+Which of the following options contain two different ways of introducing your name in Spanish?
```
```suggestion
Which of the following options contains two different ways of introducing your name in Spanish?
```

estefaniacn commented on curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c850ed23d494c9be5cb22a.md:
```diff
@@ -1,58 +1,85 @@
+dashedName: grammar-highlights
 ---
 
+<!-- GRAMMAR -->
```
I would suggest adding the `<br />` for the blank line between all the grammar highlights sections.

estefaniacn reviewed (commented):
Looks great to me, Nielda. Thank you for implementing these changes. These are two small suggestions that you may like to include.

nieldakarla committed:
add br

estefaniacn reviewed (approved):
Looks great to me. 👍 Thank you for making these updates @nieldakarla

DanielRosa74 reviewed (approved):
LGTM
